exchanges/btse/btse_wrapper.go

```diff
@@ -251,36 +266,23 @@ func (b *BTSE) UpdateTradablePairs(forceUpdate bool) error {
 
 // UpdateTicker updates and returns the ticker for a currency pair
 func (b *BTSE) UpdateTicker(p currency.Pair, assetType asset.Item) (*ticker.Price, error) {
-	if assetType == asset.Futures {
-		// Futures REST implementation needs to be done before this can be
-		// removed
-		return nil, common.ErrNotYetImplemented
-	}
-
-	fpair, err := b.FormatExchangeCurrency(p, assetType)
+	fPair, err := b.FormatExchangeCurrency(p, assetType)
 	if err != nil {
 		return nil, err
 	}
 
-	t, err := b.GetTicker(fpair.String())
+	t, err := b.GetMarketSummary(fPair.String(), assetType == asset.Spot)
```

Comment:
Last nit! This endpoint actually supports ticker batch results: https://api.btse.com/spot/api/v3.2/market_summary
So for `RESTCapabilities: protocol.Features{` could you please add `TickerBatching: true,` and then instead of requesting this with a pair, do it without and loop over the results?